assets/home/tasks-section/task-item.js

```diff
@@ -10,15 +10,21 @@ import CheckIcon from '../../icons/checked.svg';
 import ChevronRightIcon from '../../icons/chevron-right.svg';
 import { isUrlExternal } from '../utils';
 
+/**
+ * WordPress dependencies
+ */
+import { Icon, external } from '@wordpress/icons';
+
 /**
  * Tasks item component.
  *
- * @param {Object}  props       Component props.
- * @param {string}  props.title Item title.
- * @param {string}  props.url   Item URL.
- * @param {boolean} props.done  Whether item is completed.
+ * @param {Object}  props              Component props.
+ * @param {string}  props.title        Item title.
+ * @param {string}  props.url          Item URL.
+ * @param {boolean} props.done         Whether item is completed.
+ * @param {boolean} props.externalIcon Whether to show external icon.
  */
-const TaskItem = ( { title, url, done } ) => {
+const TaskItem = ( { title, url, done, externalIcon } ) => {
```

Comment:
WDYT about calling the new prop as `external` or `isExternal`? Just a small suggestion because `externalIcon` seems something that would receive an icon.

Reply:
Makes sense, Thanks, I've updated it here https://github.com/Automattic/sensei/pull/7562/commits/e7de6d6015a3966431742255ef0dbb7169311d1d